Source/Core/Core/ActionReplay.cpp

```diff
@@ -640,14 +642,15 @@ static bool ZeroCode_MemoryCopy(const u32 val_last, const ARAddr& addr, const u3
     {  // Memory Copy With Pointers Support
       LogInfo("Memory Copy With Pointers Support");
       LogInfo("--------");
-      const u32 ptr_dest = PowerPC::HostRead_U32(addr_dest);
+      const u32 ptr_dest = PowerPC::HostRead_U32(addr_dest).value_or(0);
       LogInfo("Resolved Dest Address to: {:08x}", ptr_dest);
-      const u32 ptr_src = PowerPC::HostRead_U32(addr_src);
+      const u32 ptr_src = PowerPC::HostRead_U32(addr_src).value_or(0);
       LogInfo("Resolved Src Address to: {:08x}", ptr_src);
       for (int i = 0; i < num_bytes; ++i)
       {
-        PowerPC::HostWrite_U8(PowerPC::HostRead_U8(ptr_src + i), ptr_dest + i);
-        LogInfo("Wrote {:08x} to address {:08x}", PowerPC::HostRead_U8(ptr_src + i), ptr_dest + i);
+        PowerPC::HostWrite_U8(PowerPC::HostRead_U8(ptr_src + i).value_or(0), ptr_dest + i);
```

Comment:
Thoughts on storing the result of `PowerPC::HostRead_U8(ptr_src + i).value_or(0)` in a local for use with both the `HostWrite_U8` and the `LogInfo` that follows?
Or is the overhead of calling it twice neglible with the changes in `HostRead_UX`?